### PR TITLE
Add templates for Issues and PR's

### PR DESCRIPTION
**Pull request by buger**
In order to improve quality of user contributions we need require them fill some essential fields when they create issues or PR’s. See PR content.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Do you want to request a *feature* or report a *bug*?**
```
Comment (mvdan) [inline .github/ISSUE_TEMPLATE.md]: Perhaps request a label here, like `enhancement` or `bug`?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+**What is the current behavior?**
+
+**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**
+
+**What is the expected behavior?**
```
Comment (mvdan) [inline .github/ISSUE_TEMPLATE.md]: I think this should be further up, especially before the steps to reproduce.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+*Before* submitting a pull request, please make sure the following is done...
+
+1. Fork the repo and create your branch from `master`.
```
Comment (mvdan) [inline .github/PULL_REQUEST_TEMPLATE.md]: If they're opening a PR, this is already done, isn't it?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+2. If you've added code that should be tested, add tests!
+3. If you've changed APIs, describe what needs to be updated in documentation.
```
Comment (mvdan) [inline .github/PULL_REQUEST_TEMPLATE.md]: "in the documentation"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+4. Ensure the test suite passes (`go test`).
+5. Make sure your code lints (`go fmt && go vet`).
```
Comment (mvdan) [inline .github/PULL_REQUEST_TEMPLATE.md]: These will change over time. Perhaps we should just tell them to run `util/ci-test.sh`?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (buger) [inline .github/ISSUE_TEMPLATE.md]: I think only repo "members" can assign the labels

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (buger) [inline .github/PULL_REQUEST_TEMPLATE.md]: For example, If a user is not the first-time contributor, he can create a branch based on his old work. And there are many cases like this.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (buger) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'll create separate developer guidelines which include this information. Once it is added, I updated template and link to it.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mvdan) [inline .github/ISSUE_TEMPLATE.md]: You are correct, apologies.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (mvdan) [inline .github/PULL_REQUEST_TEMPLATE.md]: Oh, then perhaps also remind them to not branch from a stale master.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+*Before* submitting a pull request, please make sure the following is done...
+
+1. Fork the repo and create your branch from a **latest** `master`.
```
Comment (mvdan) [inline .github/PULL_REQUEST_TEMPLATE.md]: "the latest"

Apologies for insisting, but this will be read many times :)

Comment (buger) [inline .github/PULL_REQUEST_TEMPLATE.md]: It's totally ok, my grammar needs heavy lifting :)

**Comment by mvdan** [comment]
LGTM, assuming you'll squash before merging

**Comment by lonelycode** [comment]
@buger Care to do the honours?

**Comment by buger** [comment]
I just wanted to get some feedback from @ilijabojanovic first, but since he starting at 23 I think we can merge it and update in future if needed.

**Pull request merged**